SDWebImage/SDWebImageDefine.m

```diff
@@ -39,14 +39,14 @@ inline CGFloat SDImageScaleFactorForKey(NSString * _Nullable key) {
                 NSString *fileScale = [NSString stringWithFormat:@"@%@x.", scaleFactor];
                 if ([key containsString:fileScale]) {
                     scale = scaleFactor.doubleValue;
-                    return scale;
+                    return MAX(scale, 1);
```

Comment:
Actually, this `scale` is from internal const array `SDImageScaleFactors()`, which can not be less than 1...

Reply:
Oh,  When enter `SDImageScaleFactorForKey()`, It's not necessary to judge whether it's less than 1 ？